src/reject.rs

```diff
@@ -157,23 +203,62 @@ impl Reject for Rejection {
     }
 
     fn into_response(self) -> ::reply::Response {
-        let code = self.status();
+        use http::header::{CONTENT_TYPE, HeaderValue};
+        use hyper::Body;
 
+        let code = self.status();
         let mut res = http::Response::default();
         *res.status_mut() = code;
+
+        match serde_json::to_vec(&self) {
```

Comment:
This may make sense in JSON APIs, but maybe less so in other cases. In other cases, it may be better to just print a plain text message.

The best case would probably be if we could convert this somehow to just `Response<Cause>`, and a solution to #4 would allow a user to automatically add converting rejections into JSON (or some other format)...